infoBooks.json

```
@@ -0,0 +1,58 @@
+[
+    {
+        "book1": {
+            "cover": "https://images.cdn2.buscalibre.com/fit-in/180x180/77/7d/777dd9c09b7ce5ce98873eecac83cc79.jpg",
+            "titleb": "Lógica de Programación",
+            "autorb": "Omar Iván Buriticá"
+        }
```

It isn't necessary to define this property `book1`, `book2` ... you can directly define `cover`, `title` and `author` inside the first object.
Looks like this:
```suggestion
        "cover": "https://images.cdn2.buscalibre.com/fit-in/180x180/77/7d/777dd9c09b7ce5ce98873eecac83cc79.jpg",
        "titleb": "Lógica de Programación",
        "autorb": "Omar Iván Buriticá"
```

---

this comment was applied, I receive your suggerent.